### DISCUSSION TITLE
Error: No "exports" main defined in .../authjs-test/node_modules/@auth/core/package.json

### DISCUSSION THREAD

**Discussion by seol-gang** [nextauthjs/next-auth#8727] [Help]
### Summary

I want to use the @auth/core module to use it on the backend.

First, I tried to create a simple backend service with express for testing purposes, but when I wrote the Auth function of the @auth/core module and executed it with nodemon, an error occurred.

First of all, most of the "No exports main defined in..." errors were answered in the community as a result of incorrect node versions.

So I've been testing by running the node version all the way back to 14~18 LTS, but I still get the error.

And since I don't get any errors when I exclude "Auth" function from the code, it appears that there is something wrong with that module.

Oh, and I've also seen some replies that suggest modifying the "exports" part of the "node_modules/@auth/core/package.json" file, but I think it's correct as currently written.
```
{
...,
"exports": {
    ".": {
      "types": "./index.d.ts",
      "import": "./index.js"
    },
    "./adapters": {
      "types": "./adapters.d.ts"
    },
    "./errors": {
      "types": "./errors.d.ts",
      "import": "./errors.js"
    },
    "./jwt": {
      "types": "./jwt.d.ts",
      "import": "./jwt.js"
    },
    "./providers": {
      "types": "./providers/index.d.ts"
    },
    "./providers/*": {
      "types": "./providers/*.d.ts",
      "import": "./providers/*.js"
    },
    "./types": {
      "types": "./types.d.ts"
    }
  },
...
}
```
Is there any way to fix this?

**Comment by RosePinkDragon** [comment]
same here tried with node 16,18,20. please any help
i have tried deleting node modules and lock files, tried npm & pnpm

**Comment by shtse8** [comment]
I have the same issue using node 21

**Comment by akhnaz** [comment]
I have similar issue..but it is more like linting error. 
Solve it by changing `eslint-import-resolver-alias` to `eslint-import-resolver-custom-alias` in my `.eslintrc.json`
